plugins/grafana-custom-plugins/grafana-dependency-plugin/.config/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG grafana_version=latest
```

Comment:
why use latest and not a verified version?

Reply:
will update to 9.1.3